tests/queries/0_stateless/02456_test_attach_partition_from_table_with_more_granular_partition_expression_data_not_split.sql

```diff
@@ -0,0 +1,13 @@
+DROP TABLE IF EXISTS source;
+DROP TABLE IF EXISTS destination;
+
+CREATE TABLE source (timestamp DateTime) engine=MergeTree ORDER BY tuple() PARTITION BY toYYYYMMDD(timestamp);
+CREATE TABLE destination (timestamp DateTime) engine=MergeTree ORDER BY tuple() PARTITION BY toYYYYMM(timestamp);
+
+INSERT INTO TABLE source VALUES ('2010-03-02 02:01:01'), ('2010-03-02 02:01:03');
+
+ALTER TABLE destination ATTACH PARTITION '20100302' FROM source;
+
+SELECT * FROM source;
+SELECT 1;
```

Comment:
Consider using `-- { echoOn }` instead, to print each query (you can put after `CREATE TABLE` I guess)

Reply:
hm.. not sure what `echoOn` does, but I would assume it simply echoes back the command issued. How would that help with printing both queries and comparing against expected output?

Reply:
As far as I can understand here you are using `SELECT 1` to distinguish two queries, but using `-- { echoOn }` before `SELECT` queries, the `.reference` file will contain the query itself, and so you will not need this `SELECT 1`, but this is minor.

Reply:
Oh, I see. That makes sense. Thanks!